go.mod

```diff
@@ -3,65 +3,65 @@ module github.com/openshift/openshift-controller-manager
 go 1.18
 
 require (
-	github.com/BurntSushi/toml v0.3.1
+	github.com/BurntSushi/toml v1.2.0
 	github.com/blang/semver/v4 v4.0.0
 	github.com/containers/image v3.0.2+incompatible
-	github.com/containers/image/v5 v5.11.0
-	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
+	github.com/containers/image/v5 v5.22.0
+	github.com/coreos/go-systemd v0.0.0-20190620071333-e64a0ec8b42a
 	github.com/davecgh/go-spew v1.1.1
 	github.com/google/go-cmp v0.5.6
 	github.com/google/gofuzz v1.2.0
-	github.com/google/uuid v1.1.2
-	github.com/hashicorp/golang-lru v0.5.1
-	github.com/openshift/api v0.0.0-20220616165336-689617d54300
-	github.com/openshift/build-machinery-go v0.0.0-20220429084610-baff9f8d23b3
-	github.com/openshift/client-go v0.0.0-20220603133046-984ee5ebedcf
-	github.com/openshift/library-go v0.0.0-20220615161831-8b2df431789c
-	github.com/openshift/runtime-utils v0.0.0-20220513161558-c736ec4e99ce
+	github.com/google/uuid v1.3.0
+	github.com/hashicorp/golang-lru v0.5.4
+	github.com/openshift/api v0.0.0-20221004120407-c46852673d03
+	github.com/openshift/build-machinery-go v0.0.0-20220913142420-e25cf57ea46d
+	github.com/openshift/client-go v0.0.0-20220915152853-9dfefb19db2e
+	github.com/openshift/library-go v0.0.0-20221004125358-0d713c6520e2
+	github.com/openshift/runtime-utils v0.0.0-20220926190846-5c488b20a19f
 	github.com/prometheus/client_golang v1.12.1
 	github.com/prometheus/client_model v0.2.0
 	github.com/spf13/cobra v1.4.0
-	k8s.io/api v0.24.0
-	k8s.io/apimachinery v0.24.0
-	k8s.io/apiserver v0.24.0
-	k8s.io/client-go v0.24.0
-	k8s.io/component-base v0.24.0
-	k8s.io/controller-manager v0.24.0
-	k8s.io/klog/v2 v2.60.1
-	k8s.io/kubectl v0.24.0
-	k8s.io/kubernetes v1.24.0
-	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/apiserver v0.25.0
+	k8s.io/client-go v0.25.0
+	k8s.io/component-base v0.25.0
+	k8s.io/controller-manager v0.25.0
+	k8s.io/klog/v2 v2.80.1
+	k8s.io/kubectl v0.25.0
+	k8s.io/kubernetes v1.25.0
+	k8s.io/utils v0.0.0-20220823124924-e9cbc92d1a73
```

Comment:
https://github.com/kubernetes/kubernetes/blob/a866cbe2e5bbaa01cfd5e969aa3e033f3282a8a2/go.mod (1.25.0 release) sets k8s.io/utils to `v0.0.0-20220728103510-ee6ede2d64ed`